helm_deploy/prisoner-content-hub-frontend/values.cw.yaml

```diff
@@ -0,0 +1,7 @@
+application:
+  config:
+    appName: "HMYOI Cookham Wood"
+    establishmentName: cookhamwood
+
+ingress:
+  domainPrefix: cw
```

Comment:
Could we use `cookhamwood` (with or without hyphens) or something more descriptive in the name, domain, and references in `config.yml` here so the URL is meaningful?

Reply:
Updated domain prefix